packages/calcite-components/src/components/time-picker/time-picker.e2e.ts

```diff
@@ -1254,127 +1182,229 @@ describe("calcite-time-picker", () => {
+              // We're leaving this conditional check here in case a new locale is added in the future that might need to test the second suffix.
+              expect(secondSuffixEl).toEqualText(expectedLocalizedSecondSuffix);
+            }
+            expect(meridiemEl).toEqualText(expectedLocalizedMeridiem);
           });
 
-          const hourEl = await page.find(`calcite-time-picker >>> .${CSS.hour}`);
-          const hourSuffixEl = await page.find(`calcite-time-picker >>> .${CSS.hourSuffix}`);
-          const minuteEl = await page.find(`calcite-time-picker >>> .${CSS.minute}`);
-          const minuteSuffixEl = await page.find(`calcite-time-picker >>> .${CSS.minuteSuffix}`);
-          const secondEl = await page.find(`calcite-time-picker >>> .${CSS.second}`);
-          const decimalSeparatorEl = await page.find(`calcite-time-picker >>> .${CSS.decimalSeparator}`);
-          const fractionalSecondEl = await page.find(`calcite-time-picker >>> .${CSS.fractionalSecond}`);
-          const secondSuffixEl = await page.find(`calcite-time-picker >>> .${CSS.secondSuffix}`);
-          const meridiemEl = await page.find(`calcite-time-picker >>> .${CSS.meridiem}`);
-
-          expect(hourEl).toEqualText(expectedLocalizedHour);
-          expect(hourSuffixEl).toEqualText(expectedLocalizedHourSuffix);
-          expect(minuteEl).toEqualText(expectedLocalizedMinute);
-          expect(minuteSuffixEl).toEqualText(expectedLocalizedMinuteSuffix);
-          expect(secondEl).toEqualText(expectedLocalizedSecond);
-          expect(decimalSeparatorEl).toEqualText(expectedLocalizedDecimalSeparator);
-          expect(fractionalSecondEl).toEqualText(expectedLocalizedFractionalSecond);
-          if (secondSuffixEl) {
-            // Bulgarian is the only locale Calcite supports that has a known suffix after the seconds.
-            // Esri i18n prefers this character be removed for short time formats, which is the only format currently that time-picker supports.
-            // We're leaving this conditional check here in case a new locale is added in the future that might need to test the second suffix.
-            expect(secondSuffixEl).toEqualText(expectedLocalizedSecondSuffix);
-          }
-          expect(meridiemEl).toEqualText(expectedLocalizedMeridiem);
+          it("always displays hour in 12 hour format when nudging and no value is set", async () => {
+            const page = await newE2EPage({
+              html: `<calcite-time-picker hour-format="12" lang="${locale}"></calcite-time-picker>`,
+            });
+            const hour = await page.find(`calcite-time-picker >>> .${CSS.hour}`);
+
+            await hour.click();
+
+            for (let i = 1; i < 24; i++) {
+              await page.keyboard.press("ArrowUp");
+              await page.waitForChanges();
+
+              expect(hour.textContent).toBe(i > 12 ? formatTimePart(i - 12) : formatTimePart(i));
+            }
+
+            await page.keyboard.press("Delete");
```

Comment:
Can we make both tests follow similar keyboard steps or why is delete needed in addition to the down arrow here? It's not quite clear from the test.

Reply:
Updating both tests to clear the hour input before starting the decrement steps (that's what the Delete is there for).  That way they're both consistent and do what the original test did.